charts/kong/templates/_helpers.tpl

```diff
@@ -494,7 +494,9 @@ The name of the service used for the ingress controller's validation webhook
   imagePullPolicy: {{ .Values.image.pullPolicy }}
   env:
   {{- include "kong.env" . | nindent 2 }}
-  command: [ "/bin/sh", "-c", "until kong start; do echo 'waiting for db'; sleep 1; done; kong stop" ]
+{{/* TODO: the rm command here is a workaround for https://github.com/Kong/kong/issues/6827
+     It should be removed once that's fixed */}}
+  command: [ "/bin/sh", "-c", "until kong start; do echo 'waiting for db'; sleep 1; done; kong stop; rm {{ .Values.env.prefix | default "/usr/local/kong" }}/stream_rpc.sock" ]
```

Comment:
If the `rm`ed file doesn't exist, `sh -c` will exit with non-zero status and therefore mark the initcontainer as failed. Or am I missing something?

Reply:
Indeed it does not. Good catch--the only CI check that used the DB also included stream listens to try and economize on CI runs. That made sense since stream listen functionality _should_ be independent of whether the instance is DB-backed or not, but that's no longer the case.